packages/core/src/components/Checkbox/__tests__/checkbox-tests.jest.tsx

```diff
@@ -39,6 +40,8 @@ type RenderHelper = {
   option3Text: string;
   option3Value: string;
   onChangeMock3: jest.MockedFunction<MockedFunction>;
+  defaultChecked?: boolean;
+  autoFocusOption2?: boolean;
```

Comment:
autoFocusOption2 ? :)

Reply:
:D, I can send just autoFocus and put it on the first option but I'm not sure it will be clear from outside.. WDYT?